website_blog_mail_server/models/res_partner.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, fields, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.model
+    def _notify_prepare_email_values(self, message):
+        mail_values = super(
+            ResPartner, self)._notify_prepare_email_values(message)
+        if message.model and message.model in ('blog.post', 'blog.blog'):
+            website_blog_mail_server_id = self.env['website'].sudo(
+            ).get_current_website().website_blog_mail_server_id
+            if website_blog_mail_server_id:
+                mail_values['mail_server_id'] = website_blog_mail_server_id.id
```

Comment:
```suggestion
            blog_mail_server_id = self.env['website'].sudo(
            ).get_current_website().blog_mail_server_id
            if blog_mail_server_id:
                mail_values['mail_server_id'] = blog_mail_server_id.id
```

Just to make the variable names shorter.